### PR TITLE
Parameter groups should support "postfix" for naming

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

File: NuGet.Config [hunk 1]
```diff
@@ -6,4 +6,7 @@
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
   </packageSources>
+  <disabledPackageSources>
+    <add key="nuget.org" value="true" />
```
Comment (yugangw-msft) [inline NuGet.Config]: expected change?

**Comment by yugangw-msft** [comment]
Could you update test scenario or at least regenerate the C# code to ensure your change does become effective. The code change itself looks good.

File: NuGet.Config (hunk 1 above)
Comment (matthchr) [inline NuGet.Config]: No.

**Comment by matthchr** [comment]
@yugangw-msft Yes, I will update this PR.

**Comment by matthchr** [comment]
I will look at the build failure and see why it failed -- looks like I need to merge and update...

**Comment by matthchr** [comment]
@yugangw-msft This is good to merge now provided you're okay with the changes and the jenkins build passes.

**Comment by brjohnstmsft** [comment]
Included with this PR was a fix for this issue: https://github.com/Azure/autorest/issues/479

**Pull request merged**